src/UIComponents/Predict.jsx

```diff
@@ -109,10 +108,12 @@ class Predict extends Component {
           </button>
         </div>
         {this.props.predictedLabel && (
-          <div>
-            <p />
-            <div>A.I. predicts:</div>
-            <div style={styles.contents}>
+          <div style={{ ...styles.contents, marginTop: 20 }}>
+            <div style={{ float: "left", width: "20%" }}>
+              <img style={{ width: "100%" }} src={aiBotBorder} />
+            </div>
+            <div style={{ float: "right", width: "75%" }}>
```

Comment:
small nit: can we extract styles and put them in constants with the others for consistency?

Reply:
Done.